subsys/bluetooth/controller/Kconfig.ll_sw_split

```diff
@@ -1036,6 +1038,21 @@ config BT_TICKER_SLOT_AGNOSTIC
 	  reservations and collision handling, and operates as a simple
 	  multi-instance programmable timer.
 
+config BT_TICKER_PREFER_START_BEFORE_STOP
```

Comment:
Is this being tested with both `y` and `n` in CI?

Reply:
Apparently there is no coverage for `BT_TICKER_SLOT_AGNOSTIC`.

@mtpr-ot should I just add `BT_TICKER_SLOT_AGNOSTIC` and `BT_TICKER_PREFER_START_BEFORE_STOP` as a build only test in `tests/bluetooth/init` samples?

Reply:
Added a prj_ctlr_ticker.conf to cover the changes in this PR.

Reply:
> Apparently there is no coverage for `BT_TICKER_SLOT_AGNOSTIC`.
> 
> @mtpr-ot should I just add `BT_TICKER_SLOT_AGNOSTIC` and `BT_TICKER_PREFER_START_BEFORE_STOP` as a build only test in `tests/bluetooth/init` samples?

That would be very nice, thanks.

Reply:
Added and CI passing now